composer.json

```diff
@@ -41,13 +41,15 @@
     "roave/security-advisories": "dev-master"
   },
   "require-dev": {
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.3",
+    "php": "^5.6|^7",
+    "brain/monkey": "^2.2",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.4",
+    "mikey179/vfsStream": "^1.6",
+    "phpunit/phpunit": "~4.8 || ~5.7.9",
     "sirbrillig/phpcs-variable-analysis": "^2.0",
+    "squizlabs/php_codesniffer": "^3.3",
```

Comment:
PHPCS 3.3 has not been released yet, and in any case, the new annotations came out in 3.2.0, so `^3.2` is the suitable constraint here, unless there's something in 3.3 that is critically required.